open_gpt/models/generation.py

```diff
@@ -183,6 +209,9 @@ def step_generate(
                 if not partially_stopped:
                     yield {
                         "generated_text": output,
+                        "past_key_values": PastKeyValuesOutput(past_key_values)
+                        if output_past_key_values
+                        else None,
                         "usage": {
```

Comment:
Please keep it in mind `step_generate` is the front-end API that will be directly called by the user in their application. Thus, it is better to return the application-related data (e.g., generated text) only.